notebooks/community/model_garden/model_garden_movinet_clip_classification.ipynb

```diff
@@ -0,0 +1,871 @@
+{
```

Comment:
<p>Note that Bayesian optimization is the Vertex AI Vizier service</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Thanks for the review! Could you please clarify a little bit more on this comment - do you suggest I should mention Vertex AI Vizier service here? Aren't all search algorithms backed by Vertex AI Vizier?</p>

Reply:
You're correct, the underlying search service for all algorithms is Vizier